CONTRIBUTING.md

```diff
@@ -124,7 +124,7 @@ $ TEST_DEBUG=true make test
 To test the code coverage, use:
 
 ```sh
-$ make test-cov
```

Comment:
```
test-ci-coverage:
	NODE_ENV=test BABEL_ENV=cov make bootstrap
	./scripts/test-cov.sh
	./node_modules/.bin/codecov -f coverage/coverage-final.json
```

This is for the CI so it runs make bootstrap which you wouldn't want to do each time you ran the coverage.

If we want to do this we should probably make a new command like `make test-cov` to only run `./scripts/test-cov.sh`